pavelib/utils/test/suites/python_suite.py

```diff
@@ -19,6 +19,7 @@ def __init__(self, *args, **kwargs):
         super(PythonTestSuite, self).__init__(*args, **kwargs)
         self.opts = kwargs
         self.disable_migrations = kwargs.get('disable_migrations', True)
+        self.disable_courseenrollment_history = kwargs.get('disable_courseenrollment_history', True)
```

Comment:
I don't think its necessary to change this file, since we're handling the logic in the pytest suite. I can't think of a workflow where this change is needed inside Jenkins.